src/index.ts

```diff
@@ -1,5 +1,32 @@
-export interface Sdk {}
+/// <reference path="../definitions/url-lib.d.ts"/>
```

Comment:
this is interesting, so this is like a require for ts definitions?

Reply:
Yep, that's my understanding. Should be able to get rid of it with #9 though